src/Logger/Http/Response.php

```diff
@@ -42,6 +42,7 @@
  */
 final class Response
 {
+    public const OK_RESPONSE_CODE = 200;
```

Comment:
what do you think if constant will have a name like `HTTP_OK` and `HTTP_CREATED` ?

Reply:
:ok_hand: